css-view-transitions-1/Overview.bs

```diff
@@ -787,6 +789,9 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	since that results in a circular dependency.
 	Therefore, the [=view transition layer=] is a sibling of all other content.
 
+	When a {{Document}}'s [=document/active view transition=]'s [=ViewTransition/phase=] is "`animating`",
```

Comment:
Should this have both `<code>` and quotes?

Reply:
(I'm not familiar with the conventions here, so just asking.)

Reply:
I don't think so. This spec uses `<code>` for syntax which can be used by an author. phase is internal UA state.